website_sale_stock_control/models/product.py

```diff
@@ -1,29 +1,38 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
-from openerp import fields, models
+from openerp import api, fields, models
+from openerp.http import request
+
+
+class WebsiteStockAvailable(models.AbstractModel):
+    _name = 'website.stock.available.mixin'
+
+    # To be inherited by other modules
+    website_qty_available = fields.Float(
+        compute='_compute_website_qty_available')
+
+    @api.multi
+    def _compute_website_qty_available(self):
+        for product in self:
+            product.website_qty_available = product.sudo().with_context(
+                force_company=request.env.user.company_id.id
+            ).qty_available
 
 
 class ProductTemplate(models.Model):
-    _inherit = 'product.template'
+    _name = 'product.template'
+    _inherit = ['product.template', 'website.stock.available.mixin']
 
+    # Called the same way as v11 website_sale_stock module
     inventory_availability = fields.Selection([
-        ('never', 'Allow buy products without control'),
-        ('always', 'Deny buy products without stock'),
-    ], string='Stock Control', default='never',
+        ('never', 'Always allow to buy products'),
+        ('always', "Don't allow to buy product without stock"),
+    ], string='Website Stock Control', default='never',
```

Comment:
So, should this be required or not? It wasn't clear last time 😕

Reply:
I think that is not necessary...